nf_core/__init__.py

```diff
@@ -3,6 +3,6 @@
 Shouldn't do much, as everything is under subcommands.
 """
 
-import pkg_resources
+import importlib.metadata as importlib_metadata
```

Comment:
Admittedly, I am newly using pkg_resources in Downloads:

`from pkg_resources import parse_version as VersionParser`

It is just something I came by when searching for suggestions online how to reliably sort semantic versions correctly.  I am open to change that, in case dropping pkg_resources is desired?

Reply:
I don't think we need to drop it, but can think about it if there's a better solution :)

Reply:
So far, I am [just aware that distutils is deprecated](https://peps.python.org/pep-0632/), which I had used initially before switching to `pkg_resources`. So is `importlib` the new thing to use (at least for tools)?